sdk/src/test/AllTests.ls

```diff
@@ -30,9 +30,11 @@ package {
      */
     public class AllTests {   
         public static function main() {
-            var result:TestResult = TestRunner.runAll((AllTests as Type).getAssembly());
-            
-            Process.exit(result.typeReport.successful ? 0 : 1);
+            TestRunner.onComplete += function(result:TestResult) {
+                
+                Process.exit(result.typeReport.successful ? 0 : 1);
+            };
+            TestRunner.runAll((AllTests as Type).getAssembly(), true);
```

Comment:
We should still probably be running one test at a time, but if you're already involving state and callbacks, it seems like the API itself should be instance-based instead of static if only to be more useful for other things where you want to use it a little differently.

I guess I just don't like the API not being usable for an undetermined amount of time after using it once :P